.github/workflows/build-test.yml

```diff
@@ -16,104 +16,93 @@ jobs:
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
 
       - name: Cache Docker layers
-        uses: actions/cache@v2
+        uses: actions/cache@v3
         with:
           path: /tmp/.backend-cache
           key: prime-backend-${{ github.sha }}
           restore-keys: |
             prime-backend-
 
       - name: "Login to OpenShift repository"
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           registry: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}
           username: ${{secrets.OPENSHIFT_TOOLS_SERVICE_ACCOUNT_USERNAME}}
           password: ${{secrets.OPENSHIFT_TOOLS_SERVICE_ACCOUNT_PASSWORD}}
 
       - name: "Login to RedHat container repository"
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with: 
           registry: ${{secrets.REDHAT_REGISTRY_URL}}
           username: ${{secrets.REDHAT_REGISTRY_SERVICE_ACCOUNT_USER}}
           password: ${{secrets.REDHAT_REGISTRY_SERVICE_ACCOUNT_AUTH_TOKEN}}
 
-      - name: "Build, tag and push .NET web API image"
-        id: build-image
-        uses: docker/build-push-action@v2
-        with:
-          context: prime-dotnet-webapi
-          push: true
-          file: prime-dotnet-webapi/openshift.dockerfile
-          tags: ${{secrets.OPENSHIFT_IMAGE_REPOSITORY}}/${{secrets.OPENSHIFT_LICENSE_PLATE}}-tools/prime-webapi-backend:test
-          build-args:
-            ASPNETCORE_ENVIRONMENT="Development"
-            ASPNETCORE_HTTPS_PORT=5001
-            ASPNETCORE_URLS="http://+:8080"
-            DB_CONNECTION_STRING="host=postgres;port=5432;database=prime;username=$PRIME_DB_USER;password=$PRIME_DB_PASSWORD"
-            DB_HOST=postgres
-            DOTNET_CLI_HOME="/tmp/DOTNET_CLI_HOME"
-            PGPASSWORD=${{secrets.PRIME_DB_PASSWORD}}
-            POSTGRESQL_ADMIN_PASSWORD=${{secrets.PRIME_DB_PASSWORD}}
-            POSTGRESQL_DATABASE=${{secrets.PRIME_DB_USER}}-${{steps.PR.outputs.number}}
-            POSTGRESQL_USERNAME=${{secrets.PRIME_DB_USER}}
-            SVC_NAME="test"
-          cache-from: type=local,src=/tmp/.backend-cache
-          cache-to: type=local,dest=/tmp/.backend-cache-new,mode=max
-
-      - name: Move cache
+      - name: "Build .NET web API image"
         run: |
-          rm -rf /tmp/.backend-cache
-          mv /tmp/.backend-cache-new /tmp/.backend-cache
+          docker build prime-dotnet-webapi/ \
+          --file prime-dotnet-webapi/openshift.dockerfile \
+          --tag prime-webapi-backend \
+          --build-arg BASPNETCORE_ENVIRONMENT="Development" \
+          --build-arg ASPNETCORE_HTTPS_PORT=5001 \
+          --build-arg ASPNETCORE_URLS="http://+:8080" \
+          --build-arg DB_CONNECTION_STRING="host=postgres;port=5432;database=prime-dev;username=$PRIME_DB_USER;password=$PRIME_DB_PASSWORD" \
```

Comment:
This should be corrected for `test` environment?

Reply:
Good catch! Thanks! I've updated it to reflect the older db connection string, also found spelling mistake and corrected.